polaris.shopify.com/src/types.ts

```diff
@@ -81,5 +81,8 @@ export interface HighlightableSearchResult {
 }
 
 export enum Breakpoints {
-  SMALL = 768,
+  Mobile = 500,
+  Tablet = 768,
+  Desktop = 1400,
+  DesktopLarge = 1600,
```

Comment:
For another day, but it would be nice to find a way to keep these in sync with the corresponding Sass variables.